callouts/python/README.md

```diff
@@ -4,6 +4,7 @@ Files using Copyright 2023 Google LLC & Apache License Version 2.0:
 
 * [service_callout.py](./extproc/service/callout_server.py)
 * [callout_tools.py](./extproc/service/callout_tools.py)
+* [command_line_tools.py](./extproc/service/command_line_tools.py)
 
 # Requirements
 
```

Comment:
In the "Quick start" section, comment "The minimal operation of this Python-based ext_proc server" needs to be updated (i.e. at least made plural) since this directory now houses two ext_proc server implementations.

Reply:
done.